students/eldonab/lesson03/mailroom.py

```diff
@@ -1,25 +1,35 @@
 #!/usr/bin/env python3
 #create a list of at least 5 donors.
 donors_list = [["John Smith", 50000, 600000], ["Bill Gates", 100000, 400, 7000], ["Andy Jones", 400000, 600000], ["Hannah Jones", 30000, 40000], ["Anna Allen", 80000,90000, 1000]] 
+
+
 # to add a new donation amount to an existing donor's name.
 def add_donation_to_list(res_name):
+    """Add a new donation amount to an existing donor's name."""
     amount = float(input("How much is the new donation amount? "))
     for donor in donors_list:
         if res_name in donor:
             donor.append(amount)
+
+
 # to add a new donor name&donation amount:
 def add_new_donor(new_donor):
+    """Add a new donor name and donation amount."""
     amount = float(input("How much is the new donation amount? "))
     donors_list.append([new_donor, amount])
     
+
 # to check whether donor already exists in the list.
 def is__in_list(donor_name):
+    """Check whether donor already exists in the list of donors."""
```

Comment:
much better! (because of your docstrings you no longer need comments above the function definition.